### PR TITLE
clean unused files in Docker Provider

### PR DESCRIPTION
**Pull request by wanlonghenry**
1. Dockerfile for linux since we are now using Dockerfile.multiarch
2. multiarch-agent-deployment  agent-deployment   in deployment pipeline
3. .pipelines/azure_pipeline_prod.yaml   .pipelines/azure_pipeline_dev.yaml

### CODE REVIEW

**Review by jatakiajanvi12 — COMMENTED** [review]
There are some references in the README file as well...Can you look at those?

**Comment by wanlonghenry** [comment]
> There are some references in the README file as well...Can you look at those?

checked updated one working for it

File: kubernetes/linux/Dockerfile.multiarch [hunk 1]
```diff
@@ -32,7 +32,7 @@ WORKDIR ${tmpdir}
 RUN chmod 775 $tmpdir/*.sh; sync; $tmpdir/setup.sh ${TARGETARCH}
 
 FROM ${MARINER_DISTROLESS_IMAGE} AS distroless_image
-MAINTAINER OMSContainers@microsoft.com
+LABEL maintainer="OMSContainers@microsoft.com"
```
Comment (jatakiajanvi12) [inline kubernetes/linux/Dockerfile.multiarch:35]: Why this change?

File: .pipelines/build-linux.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ make
 cd $DIR
 
 echo "------------ Bundle Shell Extension Scripts for Agent Release -------------------------"
-cd $DIR/../deployment/agent-deployment/ServiceGroupRoot/Scripts
+cd $DIR/../deployment/mergebranch-multiarch-agent-deployment/ServiceGroupRoot/Scripts
```
Comment (jatakiajanvi12) [inline .pipelines/build-linux.sh:19]: Not a 100% sure if we even use this file.

File: kubernetes/linux/Dockerfile.multiarch (hunk 1 above)
Comment (wanlonghenry) [reply kubernetes/linux/Dockerfile.multiarch:35]: MAINTAINER has been deprecated: https://github.com/apache/couchdb-docker/issues/126

File: .pipelines/build-linux.sh (hunk 2 above)
Comment (wanlonghenry) [reply .pipelines/build-linux.sh:19]: yes not sure neither, maybe keep it for now, will check with Ganga

File: kubernetes/linux/Dockerfile.multiarch (hunk 1 above)
Comment (ganga1980) [reply kubernetes/linux/Dockerfile.multiarch:35]: @wanlonghenry , we have usage of MAINTAINER in multiple places in this file. can we fix all other instances as well?

File: .pipelines/build-linux.sh (hunk 2 above)
Comment (ganga1980) [reply .pipelines/build-linux.sh:19]: @wanlonghenry , lets sync up on this tomorrow morning. I see you have deleted "pushAgentToAcr.sh" which is being deleted in this PR. We need to make sure this doesnt cause any regressions.

File: kubernetes/linux/Dockerfile.multiarch (hunk 1 above)
Comment (wanlonghenry) [reply kubernetes/linux/Dockerfile.multiarch:35]: done

**Review by ganga1980 — APPROVED** [review]
